code/citadel/organs/genitals_sprite_accessories.dm

```diff
@@ -27,15 +27,57 @@
 	icon_state = "tapered"
 	name = "Tapered"
 
+//Vaginas
+/datum/sprite_accessory/vagina
+	icon = 'code/citadel/icons/vagina_onmob.dmi'
+	icon_state = null
+	name = "vagina"
+	gender_specific = 0
+	color_src = "vag_color"
+	locked = 0
+
+/datum/sprite_accessory/vagina/human
+	icon_state = "human"
+	name = "Human"
+
+/datum/sprite_accessory/vagina/gaping
```

Comment:
I think I'll just remove this for now then, since this seems like it'd need it's own framework, while the wetness ones could be sort-of tacked on to the arousal framework.